spinta/backends/__init__.py

```diff
@@ -23,6 +23,25 @@ def prepare(context: Context, type: Type, backend: Backend, value: object) -> ob
     return value
 
 
+@prepare.register()
+def prepare(context: Context, type: Array, backend: Backend, value: object) -> object:
```

Comment:
Incorrect type anotations, here you clearly expect `value` to be array and also returning `array`.